include/user_output_query.hpp

```diff
@@ -0,0 +1,126 @@
+/** \brief user_filter function helps in filtering the output as per the user
+* requirement. This function takes the unformatted output files from the 
+* simulator as the input and generates formatted output for the ABP simulator
+* and all the unit tests (sender, subnet, receiver).
+*
+* During the runtime of the program, the terminal will prompt the user to
+* enter three inputs (i.e., start time, end time and component details)
+* and these inputs will be used to print specific outputs i.e., the outputs 
+* within the timeframe and components specified by the user. User can also 
+* provide multiple components with a single space between them. 
+*
+* For example, if the user inputs start time as 00:00:00:000, end time 
+* as 00:01:00:000 and component as subnet1, then the output of the program 
+* will contain message details between the above timeframe and messages 
+* only through subnet1.
+*
+* time			value	port				component
+* 00:00:20:000	1		packet_sent_out		sender1
+* 00:00:20:000	11		data_out			sender1

```

Comment:
The text is not aligned

Reply:
Corrected the mismatched alignments
Thank you.